### ISSUE TITLE
loss log

### ISSUE THREAD

**Issue by leemengxing** [senguptaumd/Background-Matting#70]
Hello, I am conducting supervised training on my own data. Can you give your training loss log，let me know. Another question ， is unsupervised training necessary？thks

**Comment by senguptaumd** [comment]
I don't think i have the loss log with me. But I observed fast convergence in the beginning and then it slows down. But you should still train the network as long as it is decreasing the loss to obtain the best result. I would suggest using a validation set to judge that convergence.

Unsupervised training is only necessary to adapt to real data because we have limited synthetic data. If you have a large dataset of alpha mattes, this is probably not so important. Without unsupervised finetuning, I would say you would get 80% there.
